### ISSUE TITLE
Allow multiple duty rotas to allow different teams to be responsible for different services

### ISSUE THREAD

**Issue by dbuxton** [arachnys/cabot#448]
It might be easier to just have additional Cabot instances to manage this, but what do we think about having > 1 rotas managed by Cabot, with each service being associated with one of the rotas?

Use case here is around monitoring support SLAs.

@frankh @JeanFred @marcushyett interested in your thoughts.

**Comment by frankh** [comment]
@dbuxton Sounds like a good idea, but not sure how we'd actually use it

Would we have multiple people on call for different things?

**Comment by dbuxton** [comment]
Yes, we want to add out-of-hours coverage for non-technical issues that are tracked via a support queue.
